alkira/resource_alkira_credential_fortinet_instance.go

```diff
@@ -25,9 +28,22 @@ func resourceAlkiraCredentialFortinetInstance() *schema.Resource {
 				Required:    true,
 			},
 			"license_key": &schema.Schema{
-				Description: "Fortinet license key.",
-				Type:        schema.TypeString,
-				Optional:    true,
+				Description: "Fortinet license key is treated as a file path by default. `license_key` " +
+					"can also be literal file contents but `license_key_is_path` must be set to false in " +
+					"this instance.",
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"license_key_is_path": &schema.Schema{
+				Description: "Indicates to the Alkira provider if the `license_key` should " +
```

Comment:
This won't be displayed very nicely in generated doc... I think we could put more details in the resource description.